docker/pause-container.go

```diff
@@ -0,0 +1,25 @@
+package docker
+
+import (
+	"fmt"
+	"regexp"
+
+	"docker.io/go-docker/api/types"
+)
+
+func FindK8SPauseContainerID(containers []types.Container, PodName, Namespace, PodUUID string) (string, error) {
+	for _, container := range containers {
+		// ex: k8s_POD_myinit_default_05ab36d8-65aa-11e8-b35e-42010af00248_0
+		pattern := fmt.Sprintf("k8s_POD_%s_%s_%s_\\d+", PodName, Namespace, PodUUID)
+		r, err := regexp.Compile(pattern)
```

Comment:
Move out the regexp to out of the for loop, we don't need to compile it every time.